das-cli/src/common/container_manager/vault_container_manager.py

```
@@ -0,0 +1,200 @@
+            )
+        except docker.errors.APIError as e:
+            raise DockerError(e.explanation)
+
+        self.wait_for_api()
+
+    def wait_for_api(self, timeout: int = 60, interval: float = 1) -> None:
+        elapsed = 0.0
+        while elapsed < timeout:
+            try:
+                self.get_status()
+                return
```

Ruff (0.16.2)

[warning] Consider moving this statement to an `else` block

(TRY300)